.github/workflows/deploy.yml

```
@@ -0,0 +1,93 @@
+#
+# Build and deploy the documentation
+#
+name: Deploy
+
+on:
+  #pull_request: # enable pull_request for testing
+  push:
+    branches:
+      - master
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    env:
+      GMT_VERSION: 6.1.1
+      GMT_DOC_VERSION: 6.1
+    defaults:
+      run:
+        shell: bash -l {0}
+    steps:
+      # Cancel previous runs that are not completed
+      - name: Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.9.0
+
+      - name: Checkout
+        uses: actions/checkout@v2.3.4
+
+      - name: Setup Miniconda
+        uses: conda-incubator/setup-miniconda@v2.1.1
+        with:
+          python-version: 3.8
+          channels: conda-forge
+          miniconda-version: "latest"
+
+      - name: Install packages
+        run: |
+          conda install gmt=${GMT_VERSION}
+          pip install -r requirements.txt
+
+      # Download cached remote files (artifacts) from GitHub
+      - name: Download remote data from GitHub
+        uses: dawidd6/action-download-artifact@v2.13.0
+        with:
+          workflow: cache-data.yml
+          workflow_conclusion: success
+          name: gmt-cache
+          path: .gmt
+
+      # Move downloaded files to ~/.gmt directory and list them
+      - name: Move and list downloaded remote files
+        run: |
+          mkdir -p ~/.gmt
+          mv .gmt/* ~/.gmt
+          # Change modification times of the two files, so GMT won't refresh it
+          touch ~/.gmt/server/gmt_data_server.txt ~/.gmt/server/gmt_hash_server.txt
+          ls -lhR ~/.gmt
+
+      - name: Build HTML
+        run: make build_html
+
+      - name: Install TinyTeX
+        uses: r-lib/actions/setup-tinytex@v1
+
+      - name: Install LaTeX packages
+        run: |
+          tlmgr install ctex ms xecjk ulem zhnumber fancyhdr titlesec tabulary varwidth \
+                        float wrapfig parskip capt-of needspace xcolor enumitem xindy fandol \
+                        pdftexcmds upquote oberdiek caption everysel
+
+      - name: Build PDF
+        run: make build_pdf
+
+      - name: Checkout the gh-pages branch
+        uses: actions/checkout@v2.3.4
+        with:
+          ref: gh-pages
+          path: deploy
+          fetch-depth: 0
+
+      - name: Push the documentation to gh-pages
+        run: |
+          cd deploy
+          touch .nojekyll
+          rm -rvf ${GMT_DOC_VERSION}
+          cp -Rvf ../build/dirhtml/ ${GMT_DOC_VERSION}/
```

We usually use `-r` instead of `-R`, right?
```suggestion
          cp -rvf ../build/dirhtml/ ${GMT_DOC_VERSION}/
```